### PR TITLE
Done: added homework of the lesson 04

### PR DESCRIPTION
**Pull request by max3koz**
Added homework of the lesson 04

### CODE REVIEW

File: lesson_04/homework_04.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+hour after hour. And when the middle of the afternoon came, from being a
+poor poverty, stricken boy in the .... morning, Tom was literally
+rolling in wealth."""
+
+##  ПЕРЕЗАПИСУЙТЕ зміст змінної adwentures_of_tom_sawer у завданнях 1-3
+# task 01 ==
+""" Дані у строці adwentures_of_tom_sawer розбиті випадковим чином, через помилку.
+треба замінити кінець абзацу на пробіл .replace("\n", " ")"""
+print("Task 01")
+adwentures_of_tom_sawer = adwentures_of_tom_sawer.replace("\n", " ")
```
Comment (DartBrovsky) [inline lesson_04/homework_04.py]: It is better to initialize new variable with name, not reassign old one with actions with it. 
For example - adwentures_of_tom_sawer_replaced = adwentures_of_tom_sawer.replace("\n", " ")

File: lesson_04/homework_04.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+print("\nTask 03")
+adwentures_of_tom_sawer = " ".join(adwentures_of_tom_sawer.split())
+print(adwentures_of_tom_sawer)
+
+# task 04
+""" Виведіть, скількі разів у тексті зустрічається літера "h"
+"""
+print("\nTask 04")
+count = adwentures_of_tom_sawer.find("h")
+print(f"There are {count} times the letter \"h\" in the sentence \"adwentures_of_tom_sawer\".")
```
Comment (DartBrovsky) [inline lesson_04/homework_04.py:52]: count variable value will be the index of it`s existence in the string. If you want to count how much time "h" symbol is present in the string - you need to iterate for whole string and each time if symbol equal to "h" - increase counter by 1.

File: lesson_04/homework_04.py [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+adwentures_of_tom_sawer_sentences = None
+adwentures_of_tom_sawer_sentences = adwentures_of_tom_sawer.split(". ")
+print(adwentures_of_tom_sawer_sentences)
+
+# task 08
+""" Виведіть четверте речення з adwentures_of_tom_sawer_sentences.
+Перетворіть рядок у нижній регістр.
+"""
+print("\nTask 08")
+print(adwentures_of_tom_sawer_sentences[3].lower())
```
Comment (DartBrovsky) [inline lesson_04/homework_04.py]: Here you need to do 2 actions. First - print 4th symbol, 2nd to do lower for whole string. In your solution you print only 4th symbol in lower registry.

**Review by DartBrovsky — CHANGES_REQUESTED** [review]
Overall your solution is good. You have several small mistakes. Please take a look into them with my comments and push fixes. Thank you. 
You done a good job).

File: lesson_04/homework_04.py (hunk 1 above)
Comment (max3koz) [reply lesson_04/homework_04.py]: Fixed in a562dda4dc7cfd297643dcf1f5dff66a9ac76b0f

File: lesson_04/homework_04.py (hunk 2 above)
Comment (max3koz) [reply lesson_04/homework_04.py:52]: Fixed ib a562dda4dc7cfd297643dcf1f5dff66a9ac76b0f

File: lesson_04/homework_04.py (hunk 3 above)
Comment (max3koz) [reply lesson_04/homework_04.py]: Fixed in a562dda4dc7cfd297643dcf1f5dff66a9ac76b0f

**Review by DartBrovsky — APPROVED** [review]

**Pull request merged**